docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[elastic-agent-input-settings]]
+== Input settings
+
+Input settings specify how {agent} locates and processes input data.
+
+You can specify multiple inputs, and you can specify the same input type more
+than once. For example:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+- type: log
+  paths:
```

Comment:
@ph I remember that this is not working and it has to go under `streams` currently?

Reply:
Yes exactly.

Reply:
All of these examples are simply copied from the Beats documentation and need to be modified to follow the new agent syntax. I need help with that, though, because the examples should be reviewed and validated, too. Rather than adding comments, it's probably better for you to pull down the branch and then push the example changes to my fork. I think it's OK if we don't test everything for the first iteration, but we should have a plan to test all of the documented configs before GA. If there's something repetitive that needs to be changed, I can ask other folks on the doc team to help out.

(I had know idea we had SO MANY input settings in Filebeat. Some of the descriptions needed a lot of love, too, so this has been a lot more work than I anticipated.)